Makefile

```diff
@@ -133,6 +133,18 @@ test-unit-js:
 		yarn test \
 	'
 
+# Currently the e2e script is git initializer-centric
+#
+# Relies on `make hack` being run and/or the local kube context pointing
+# to a k8s cluster with Brigade running
+.PHONY: test-e2e
+test-e2e:
+	@kubectl --namespace brigade port-forward service/brigade-apiserver 7000:443 &>/dev/null & \
+		echo $$! > /tmp/brigade-apiserver.PID
```

Comment:
Something looks suspicious here. The previous line doesn't end with `&&`, so how does this line run?

Reply:
Good eye.  It works b/c the background operator `&` can chain commands:
```
 $ echo "foo" & echo "bar"
[1] 71188
bar
foo
[1]+  Done                    echo "foo"
```
Here the second command just consumes the process ID created from the first (backgrounded) command and echos it to a file.

Reply:
TIL that the background operator can also chain commands.